tests/loki/iofq_test.py

```diff
@@ -175,11 +176,49 @@ def test_pipeline_can_compute_IofQ_merging_events_from_multiple_runs():
     pipeline.set_param_series(Filename[SampleRun], sample_runs)
     pipeline.set_param_series(Filename[BackgroundRun], background_runs)
 
-    pipeline.insert(sans.i_of_q.merge_multiple_runs)
+    pipeline.insert(sans.merge_runs)
     result = pipeline.compute(BackgroundSubtractedIofQ)
     assert result.dims == ('Q',)
 
 
+def test_pipeline_can_compute_IofQ_merging_events_from_banks():
+    params = make_params()
+    del params[NeXusDetectorName]
+
+    pipeline = sciline.Pipeline(loki_providers_no_beam_center_finder(), params=params)
+    pipeline[BeamCenter] = _compute_beam_center()
+    pipeline.set_param_series(PixelMaskFilename, ['mask_new_July2022.xml'])
+    pipeline.set_param_series(NeXusDetectorName, ['larmor_detector'])
```

Comment:
Can we also use e.g. twice the same bank, as I believe for now, only the
```Py
if len(data) == 1:
    return data[0]
```
code branch is being tested, which is a special case?

Reply:
Unfortunately we cannot, since for a param series the index is the same as the values, and the index must be unique. We could change everything to a param table, introducing an extra type for the row index.

Note that the other branch (of the helper function) is tested by the run-merge. But I agree that this is not ideal. I will try to write a unit test just for the merge function.